packages/playground/src/utils/date.ts

```diff
@@ -1,5 +1,15 @@
 import moment from "moment";
 
+import type { Date } from "./types";
 export default function toHumanDate(timeInSeconds: number): string {
   return moment(timeInSeconds * 1000).format("DD MMM YYYY, hh:mm A");
 }
+
+export function isReleasedOverMon(date: Date): boolean {
+  const releaseDate = new Date(date.year, date.month - 1, date.day);
```

Comment:
why do you re-assign the release date?
what if item.releaseDate is undefined as it is optional? Please wrap in try and catch

Reply:
I'm not reassigning it it's just a variable name. and it's inside if condition in the template

Reply:
I don't understand this logic. Specially subtracting 1 from the month of the passed date?


The function should be (differenceIsOverAMonth(Date1, Date2)) and when used you pass the current date and the release date